content/features/featuresDeepDive/physics/v2/compounds.md

```diff
@@ -38,3 +38,8 @@ parentShape.addChildFromParent(myMesh, sphereShape, headNode);
 const body = new BABYLON.PhysicsBody(myMesh, scene);
 body.shape = parentShape;
 ```
+The following Playground shows a simple example of creating a Container body to approximate a mesh:
+<Playground id="3H3DLR" description="Simple Physics Container example" />

```

Comment:
The `title` property is missing:

![image](https://user-images.githubusercontent.com/4152247/235164149-c6cb8733-6ba7-4332-9c76-64e10b406a91.png)

Also, there's no preview screenshot:

![image](https://user-images.githubusercontent.com/4152247/235164214-c5ce06a2-c7fa-46b3-a4ea-b25a8cafda74.png)